catalogs/catalog-lakehouse-iceberg/src/test/java/com/datastrato/gravitino/catalog/lakehouse/iceberg/integration/test/CatalogIcebergBaseIT.java

```diff
@@ -61,26 +60,29 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 import org.apache.commons.lang3.StringUtils;
-import org.apache.hadoop.hdfs.HdfsConfiguration;
 import org.apache.iceberg.NullOrder;
 import org.apache.iceberg.PartitionField;
 import org.apache.iceberg.PartitionSpec;
 import org.apache.iceberg.SortField;
+import org.apache.iceberg.catalog.SupportsNamespaces;
 import org.apache.iceberg.catalog.TableIdentifier;
 import org.apache.iceberg.exceptions.NoSuchNamespaceException;
-import org.apache.iceberg.hive.HiveCatalog;
 import org.apache.spark.sql.Dataset;
 import org.apache.spark.sql.Row;
 import org.apache.spark.sql.SparkSession;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.BeforeAll;
-import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
 
-@Tag("gravitino-docker-it")
-public class CatalogIcebergIT extends AbstractIT {
+public abstract class CatalogIcebergBaseIT extends AbstractIT {
+
+  protected static final ContainerSuite containerSuite = ContainerSuite.getInstance();
+  protected static String WAREHOUSE;
```

Comment:
could you make it non static?

Reply:
noticed most varaibles are static, could you try make them non static?  since they may shared by `IcebergHiveIT` and `IcebergRestIT`

Reply:
ok